### PR TITLE
feat(sms): Check whether SMS is enabled for the user.

### PR DESCRIPTION
**Pull request by shane-tomlinson**
@vladikoff, @philbooth - r?

### CODE REVIEW

File: send-sms-install-link/send-sms-install-link.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    var email = subject.account.get('email') || '';
+    if (/@softvision\.com$/.test(email)) {
+      return true;
+    } else if (/@mozilla\.(com|org)$/.test(email)) {
+      return true;
+    } else {
+      return false;
+    }
+
+    return true;
```
Comment (philbooth) [inline send-sms-install-link/send-sms-install-link.js]: Can this `return true` ever be hit? The `else` above handles everything right?

**Comment by philbooth** [comment]
@shane-tomlinson, I'm not going to pretend I understand how this repo works, but I've left a comment inline anyway. I'l leave @vladikoff to give the final 👍.

File: send-sms-install-link/send-sms-install-link.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+module.exports = {
+  name: 'Should the user see the "Send install link via SMS" screen',
+  hypothesis: 'Allowing users to send a Firefox Mobile link via SMS will result in a better conversion rate',
+  startDate: '2017-01-01',
+  subjectAttributes: ['account', 'uniqueUserId'],
```
Comment (vladikoff) [inline send-sms-install-link/send-sms-install-link.js]: `uniqueUserId` is not used

File: send-sms-install-link/send-sms-install-link.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  independentVariables: ['sendSms'],
+  eligibilityFunction: function (subject) {
+    var email = subject.account.get('email') || '';
```
Comment (vladikoff) [inline send-sms-install-link/send-sms-install-link.js]: you might want to check `subject.account` is defined

File: send-sms-install-link/send-sms-install-link.js (hunk 1 above)
Comment (vladikoff) [reply send-sms-install-link/send-sms-install-link.js]: yea this can also be cleaned up

File: send-sms-install-link/send-sms-install-link.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    } else {
+      return false;
+    }
+
+    return true;
+  },
+
+  groupingFunction: function (subject) {
+    return {
+      sendSms: 'treatment'
```
Comment (vladikoff) [inline send-sms-install-link/send-sms-install-link.js]: 👍

**Comment by shane-tomlinson** [comment]
@vladikoff, @philbooth - updated! Thanks for the review!

**Pull request merged**